### ISSUE TITLE
Aortic ectasia

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10316]
**Preferred term label:**
Aortic dilatation  (See issue #10315)

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 20233780 2010 ACCF/AHA/AATS/ACR/ASA/SCA/SCAI/SIR/STS/SVM Guidelines for the Diagnosis and Management of Patients With Thoracic Aortic Disease
Aneurysm (or true aneurysm): a permanent localized dilatation of an artery, having at least a 50% increase in diameter compared with the expected normal diameter of the artery in question. Although all 3 layers (intima, media, and adventitia) may be present, the intima and media in large aneurysms may be so attenuated that in some sections of the wall they are undetectable.
**Ectasia**: arterial dilatation less than 150% of normal arterial diameter.

PMID: 34295712 "The accepted definition for an aneurysm is a permanent focal dilatation of an artery with a ≥50% increase in diameter compared to the expected normal diameter (2,15). **Ectasia** is defined as arterial dilatation <150% of the normal arterial diameter, while arteriomegaly is diffuse arterial dilatation involving several arterial segments, with an increase in diameter >50% compared to expected. Aortic dilatation is a broader term which encompasses ectasia and aneurysm. True thoracic aortic aneurysms involve all three layers of the aortic wall; 60% affect the aortic root and/or ascending aorta, 40% the descending aorta, 10% the arch, and 10% the thoracoabdominal aorta. Aneurysms may be fusiform or saccular in morphology"

doi: 10.1016/j.ijcha.2015.01.009 "Published data on arteries diameter in healthy population are often scant or variable because of different imaging modalities used for measurement. Nevertheless, by common convention, aortic dilatation refers to a dimension that is greater than the 95th percentile for the normal person age, sex and body size. In contrast, an aneurysm is defined as a localized dilation of the aorta that is more than 50% of predicted (ratio of observed to expected diameter ≥1.5). Aneurysm should be distinguished from **ectasia**, which represents a diffuse dilation of the aorta less than 50% of normal aorta diameter."

**Parent term (use hpo.jax.org/app)**
Aortic dilatation  (See issue #10315)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:60030  Loeys-Dietz syndrome
ORPHA:558   Marfan syndrome
ORPHA:881   Turner syndrome

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
Same comment as for https://github.com/obophenotype/human-phenotype-ontology/issues/10315
